dev/com.ibm.ws.jca_fat_bval/publish/servers/com.ibm.ws.jca.fat.bval/bootstrap.properties

```diff
@@ -0,0 +1,25 @@
+###############################################################################
+# Copyright (c) 2014, 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+com.ibm.ws.logging.trace.specification=*=event=enabled:\
```

Comment:
this trace spec is pretty heavy. Lets take out webcontainer, EJBContainer, and Transaction